lib/.eslintrc

```diff
@@ -0,0 +1,13 @@
+{
+  "parserOptions": {
+      "ecmaVersion": 5
+  },
+  "extends": "eslint:recommended",
+  "rules": {
+    "no-undef": "warn",
+    "no-unused-vars": "warn"
```

Comment:
How to avoid warnings from bindings functions like `Ref_current` that are exported from a `.js` file?